### PR TITLE
refactor(calendar-view): [calendar-view] refactor theme vars

### PR DESCRIPTION
**Pull request by Youyou-smiles**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
	- Enhanced styling and layout for the calendar view, improving visual hierarchy and consistency.
	- Introduction of new CSS variables for better maintainability and flexibility in styling.

- **Bug Fixes**
	- Updated test case for the calendar's disabled day selection to ensure accurate element targeting.

- **Refactor**
	- Simplified the structure of the calendar view component's header for improved layout. 

- **Chores**
	- Removed outdated properties from the old theme definition to streamline the styling process.

### CODE REVIEW

File: packages/theme/src/calendar-view/index.less [hunk 1]
```diff
@@ -17,80 +17,68 @@
 @calendar-view-prefix-cls: ~'@{css-prefix}calendar-view';
 
 .@{calendar-view-prefix-cls} {
-  .component-css-vars-calendar-view();
-
+  .inject-CalendarView-vars();
   width: 100%;
   height: auto;
+  padding-top: 32px;
+  border: 1px solid #EBEBEB;
```
Comment (zzcr) [inline packages/theme/src/calendar-view/index.less:24]: border颜色需要提取成变量

File: packages/theme/src/calendar-view/index.less [hunk 2]
```diff
@@ -17,80 +17,68 @@
...
-  }
 
   &__radio {
     flex-shrink: 0;
-
-    .fill-brand{
-      fill:#fff;
+    .tiny-radio-button__inner {
+      height: 32px;
+      background-color: rgba(0, 0, 0, 0.05);
```
Comment (zzcr) [inline packages/theme/src/calendar-view/index.less]: 这个颜色值也需要提取变量

File: packages/theme/src/calendar-view/index.less [hunk 3]
```diff
@@ -104,66 +92,58 @@
...
-          fill: var(--ti-calendar-view-header-right-svg-fill);
+          fill: var(--tv-CalendarView-header-svg-fill);
         }
       }
     }
 
     &__main {
       width: 100%;
       overflow: hidden;
+      border-top: 1px solid #EBEBEB;
```
Comment (zzcr) [inline packages/theme/src/calendar-view/index.less:118]: 颜色值需要提取变量

File: packages/theme/src/calendar-view/index.less [hunk 4]
```diff
@@ -599,17 +542,28 @@
 
       >li{
         height: 100%;
-        padding: 12px 8px;
+        padding: var(--tv-CalendarView-week-li-padding-vertical) var(--tv-CalendarView-week-li-padding-horizontal);
```
Comment (zzcr) [inline packages/theme/src/calendar-view/index.less]: vertical改成y horizontal改成x

File: packages/theme/src/calendar-view/index.less (hunk 1 above)
Comment (Youyou-smiles) [reply packages/theme/src/calendar-view/index.less:24]: > border颜色需要提取成变量

vars.less文件里没有#EBEBEB

**Review by zzcr — APPROVED** [review]

**Pull request merged**